src/branch_view.js

```diff
@@ -55,9 +55,12 @@ function renderHtmlBlob(parent, html, hasApp) {
 
 	banner_utils.addClass(banner, 'branch-banner-is-active');
 
-	setTimeout(function() {
-			banner.style.top = '0';
-	}, 100);
+	if(parent === document.body) {
+		parent.insertBefore(banner, parent.firstChild);
+	}
+	else {
+		parent.appendChild(banner);
```

Comment:
will this conflict w/ https://github.com/BranchMetrics/web-branch-deep-linking/pull/348/files#diff-d27a3ecb19b60744268743ff1db60a6eR54